src/Stratis.Bitcoin.Features.Wallet/Broadcasting/FullNodeBroadcasterManager.cs

```diff
@@ -29,8 +29,22 @@ public FullNodeBroadcasterManager(IConnectionManager connectionManager, IMempool
                 return;
 
             var state = new MempoolValidationState(false);
+
             if (!await this.mempoolValidator.AcceptToMemoryPool(state, transaction).ConfigureAwait(false))
-                this.AddOrUpdate(transaction, State.CantBroadcast);
+            {
+                var errorMessage = string.Empty;
```

Comment:
can be just `string errorMessage;`

Reply:
errorMessage is used a couple of lines down, so needs to be assigned at creation.